README.md

```diff
@@ -92,16 +92,16 @@ To do this on Windows, you can run:  `$env:sendbirdkey="<SECRET>"; node server.j
 
 ## Mobile App API Surface
 
-__Endpoint__: `/token/:fedToken/:authType`
+__Endpoint__: `/token/<facebook||google>?access_token=<token>`
```

Comment:
It looks like this might have been refactored out. Is there still a way for the client to get an ID token? The primary reasons we need the ID are: 

1. Tell whether the user is a "mentor" or "mentee", which changes the client-side behavior 
2. Keep track of the federated token

It looks like your changes handle 2, but I'm not sure if 1 is still supported as needed

Reply:
I changed the token endpoint to the register endpoint as i thought it was more appropriate given the flow considered above. This endpoint returns a user id and api key like this:
```
{
    "api_key": "230d7d115dd1401eb1e486d41cb80129",
    "user_id": "5c15c6abbf35ae40576316m9"
}
```

Reply:
I think it makes sense to treat it as "register" meaning to register the user for an API key (essentially a long-lived session secret). I think the two changes we want though is for there to be the `user_type` included in this response (so that the client knows if it should show "mentor" or "mentee" behavior), and for this to only return an API key in the case where the user is already known to us. In the future, we will want to support the ability for new users to receive an API key, but we're not sure yet how that will look.